src/storyEngine/storyNodes/StoryNode.java

```diff
@@ -31,6 +31,9 @@
 
 	@Element(name="teaserText")
 	protected String m_teaserText;
+	
+	@Element(name="teaserImage")

```

Comment:
Make it `required=false` and check the constructors so they match...then you can remove the `none` teaser image tags.